### DISCUSSION TITLE
When will be the next release?

### DISCUSSION THREAD

**Discussion by qqlearn123** [duckdb/duckdb#8546] [General]
Would love to use latest features/optimizations.

**Comment by l1t1** [comment]
you can download build zip files from https://github.com/duckdb/duckdb/actions

or see
https://duckdb.org/docs/installation/

**Reply by qqlearn123** [reply]
Thanks.

I am more of building a lib on top of duckdb. Not sure how I can leverage other installation methods. 

If duckdb can release versions periodically, I can just update dependency version number in the setup.py file.

**Reply by Mause** [reply]
Documentation on installing nightly builds is available here: https://duckdb.org/docs/installation/

For Python, we provide prereleases, which are published to pypi

**Comment by l1t1** [comment]
https://github.com/duckdb/duckdb/issues/8531#issuecomment-1675357926 says version 0.9 will be released on September.

**Reply by Mause** [reply]
Have a look at https://duckdb.org/dev/release-dates

**Comment by l1t1** [comment]
the link contains a Google link that can't visit, please post a picture.

**Comment by l1t1** [comment]
网页无法打开
位于 https://calendar.google.com/calendar/embed?height=200&wkst=2&bgcolor=%23ffffff&ctz=Europe%2FAmsterdam&mode=AGENDA&showCalendars=0&showTz=0&showTabs=1&showPrint=0&showDate=1&showNav=1&showTitle=1&title=DuckDB%20Release%20Calendar&src=Y19ycWo2MGhlbmZudWluNWtsYmF0aTZnOWtmZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t&color=%23795548 的网页无法加载，因为：

net::ERR_CONNECTION_ABORTED

**Reply by Mause** [reply]
I take it Google calendar is blocked in your country?

![Screenshot_20230813-144450.png](https://github.com/duckdb/duckdb/assets/1405026/b0969b92-61ad-4584-8588-7fc20fa607fd)

**Comment by l1t1** [comment]
will version 0.9.0 be released today?

**Reply by Mause** [reply]
The release date has been updated to the 18th of September, but we may change it again to make time for more testing

**Reply by szarnyasg** [reply]
@l1t1 We keep the page https://duckdb.org/dev/release-dates.html up-to-date with this information.

**Reply by l1t1** [reply]
I cannot visit google, so I asked here

**Comment by arjenpdevries** [comment]
Would be a great WASM DuckDB demo to query the calendar from a DUCKDB database located on a public URL :-)

**Reply by carlopi** [reply]
Just read this, and it's indeed a very cool demo.

In DuckDB CLI this works:
```sql
SELECT summary,date FROM (
    SELECT unnest(items, recursive := TRUE) FROM 
        read_json_auto( 'https://www.googleapis.com/calendar/v3/calendars/c_rqj60henfnuin5klbati6g9kfg@group.calendar.google.com/events?key=INSERT_YOUR_GOOGLE_API_TOKEN' )
) ORDER BY date;
```
and gives:
```
┌───────────────────────┬────────────┐
│        summary        │    date    │
│        varchar        │    date    │
├───────────────────────┼────────────┤
│ DuckDB Release        │ 2021-11-15 │
│ DuckDB Feature Freeze │ 2021-11-16 │
│ Feature Freeze        │ 2022-01-31 │
│ DuckDB Release        │ 2022-02-07 │
│ Feature Freeze        │ 2022-02-08 │
│ Feature Freeze        │ 2022-03-28 │
│ Feature Freeze        │ 2022-04-04 │
│ Release 0.3.3         │ 2022-04-06 │
│ Feature Freeze        │ 2022-08-29 │
│ Release 0.5.0         │ 2022-09-05 │
│ Feature Freeze        │ 2022-09-06 │
│ Feature Freeze        │ 2022-11-07 │
│ Release 0.6.0         │ 2022-11-14 │
│ Feature Freeze        │ 2022-11-15 │
│ Bugfix Release 0.6.1  │ 2022-12-06 │
│ Feature Freeze        │ 2023-02-06 │
│ Release 0.7.0         │ 2023-02-13 │
│ Release 0.7.1         │ 2023-02-27 │
│ Feature Freeze        │ 2023-05-08 │
│ Release 0.8.0         │ 2023-05-17 │
│ Release 0.8.1         │ 2023-06-13 │
│ Release 0.9.0         │ 2023-09-26 │
├───────────────────────┴────────────┤
│ 22 rows                  2 columns │
└────────────────────────────────────┘
```

This will NOT work, as of now, in https://shell.duckdb.org, trying to find out what goes wrong.

---
Thanks a lot, keep these idea coming!

**Reply by carlopi** [reply]
Now also fixed in duckdb-wasm (needs an hour or so for the build to go live though).

**Comment by l1t1** [comment]
will version 0.9 released tomorrow?

**Reply by darthf1** [reply]
Tuesday 26th of sept

**Comment by l1t1** [comment]
when will version 0.9.1 be  released?

**Reply by Mause** [reply]
We just released 0.9.0, why are you asking about 0.9.1?

**Reply by l1t1** [reply]
as some issues I concerned are still open.

**Reply by Mause** [reply]
For the past couple of releases, we've made patch releases to fix high severity bugs found after release. I'm not aware of any plans to make a 0.9.1 release as yet

**Reply by l1t1** [reply]
got it

**Comment by l1t1** [comment]
what is the schedule now?

**Reply by Giorgi** [reply]
Jan 29

**Reply by l1t1** [reply]
where to see the duckdb roadmap? like https://github.com/ClickHouse/ClickHouse/issues/44767

**Reply by Mause** [reply]
> where to see the duckdb roadmap? like https://github.com/ClickHouse/ClickHouse/issues/44767

There is no public roadmap

**Comment by szarnyasg** [comment]
The next release, v0.9.2, is scheduled for next Monday, see:
https://duckdb.org/dev/release-dates.html

**Comment by l1t1** [comment]
why 0.9.2 binary is smaller than 0.9.1?
0.9.1
-rwxr-xr-x 1 u0_a200 u0_a200 43714576 Oct 11 19:18 duckdb
0.9.2
-rwxr-xr-x 1 root root 38958616 Nov 14 08:42 duckdb

**Comment by l1t1** [comment]
what are the new features of the next release?

**Comment by l1t1** [comment]
it's useful to add something like https://www.lua.org/versions.html to the documents
